From 29e171ef82bed2b173b762d81fdf5db5f0e3247a Mon Sep 17 00:00:00 2001
From: Christian Fetzer <fetzer.ch@gmail.com>
Date: Fri, 23 Nov 2018 19:19:35 +0100
Subject: [PATCH 1/1] patch: Install the built library

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1b3459a5..12c4ff2d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -807,3 +807,7 @@ list(APPEND CPACK_SOURCE_IGNORE_FILES "${CMAKE_BINARY_DIR}")
 # CPack must be included after the CPACK_* variables are set in order for those
 # variables to take effect.
 Include(CPack)
+
+# install the generated shared library
+install(FILES "$<TARGET_FILE:dolphin_libretro>"
+        DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/libretro)
-- 
2.17.1

